client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -57,14 +57,16 @@ $.entwine('ss', function($){
       ignoreFieldSelector: '.no-change-track, .ss-upload :input, .cms-navigator :input'
     },
 
-
     /**
-     * Variable: ValidationErrorShown
      * Boolean for tracking whether a validation error has been already been shown. Used because tabs can
-     * sometimes be inadvertently initialised multiple times, but we don't want duplicate messages
-     * (Boolean)
+     * sometimes be inadvertently initialised multiple times, but we don't want duplicate messages.
      */
-    ValidationErrorShown: false,
+    getValidationErrorShown: function() {
+      return Boolean(this.data('_validationErrorShown'));
+    },
+    setValidationErrorShown: function(value) {
+      this.data('_validationErrorShown', value);
+    },
```

Comment:
Turning this from an entwine property into regular javascript data means it is tied to the actual DOM element, which means we can rely on it being reset when a new DOM element is added.